mh/pat.py

```diff
@@ -806,9 +806,9 @@ def sendAnsServerTime(self, converted_country_code, seq):
         JP: サーバ時刻返答
         TR: Server time response
         """
-        dummy_1 = 0  # in-game time?
-        current_time = time_utils.datetime_to_int(datetime.now())
-        data = struct.pack(">II", dummy_1, current_time)
+        server_time = time_utils.current_server_time()
+        current_time = time_utils.current_server_time()  # Epoch time
```

Comment:
Please add a comment explaining why this values are the same but means different things.

Reply:
Done